### PR TITLE
Add list credential functionality with frontend search

### PR DESCRIPTION
**Pull request by mattdean-digicatapult**
Search for issued certificates by flyerId.
Screenshot 2022-03-28 at 16 50 11

### CODE REVIEW

File: src/components/AgentAuthority/ColumnLeft/ColumnLeftWrap/ColumnLeftWrap.js [hunk 1]
```diff
@@ -1,22 +1,99 @@
...
+                placeholder="GBR-RP-123456"
+              />
+              <div className="container py-3">
+                <div className="accordion" id="accordion">
+                  {renderedRecords.length === 0 && (
+                    <p className="small">No credentials found</p>
+                  )}
+
+                  {renderedRecords.length > 0 && (
+                    <>
```
Comment (n3op2) [inline src/components/AgentAuthority/ColumnLeft/ColumnLeftWrap/ColumnLeftWrap.js:55]: does it need to be wrapped in `<Framgment>`?

Comment (n3op2) [reply src/components/AgentAuthority/ColumnLeft/ColumnLeftWrap/ColumnLeftWrap.js:55]: Also it repeats, maybe we could do function?


```js
renderCard (renderRecords) => renderRecords.length > 0 ? <div className='card m0 p0'></div> : null;
```

**Review by n3op2 — APPROVED** [review]

**Review by jonmattgray — APPROVED** [review]

File: src/components/AgentAuthority/ColumnLeft/ColumnLeftWrap/ColumnLeftWrap.js (hunk 1 above)
Comment (mattdean-digicatapult) [reply src/components/AgentAuthority/ColumnLeft/ColumnLeftWrap/ColumnLeftWrap.js:55]: `<>` is shorthand for a fragment

Comment (mattdean-digicatapult) [reply src/components/AgentAuthority/ColumnLeft/ColumnLeftWrap/ColumnLeftWrap.js:55]: and the repeat was already there I believe

**Pull request merged**